bedrock/firefox/templates/firefox/home/index-master.html

```diff
@@ -0,0 +1,284 @@
+        ga_title='Trackers',
+        media_after=true
+      ) %}
+
+        <div class="mzp-c-menu-list mzp-t-cta mzp-t-download">
+          <h4 class="mzp-c-menu-list-title">{{ get_browser }}</h4>
+          <ul class="mzp-c-menu-list-list" id="menu-browsers">
+            <li class="mzp-c-menu-list-item"><a href="{{ url('firefox.download.thanks') }}">{{ get_desktop }}</a></li>
+            <li class="mzp-c-menu-list-item"><a href="{{ settings.GOOGLE_PLAY_FIREFOX_LINK_UTMS }}" rel="external noopener" data-cta-type="link" data-cta-text="Download Browser for Android"> {{ get_android }}</a></li>
+            <li class="mzp-c-menu-list-item"><a href="{{ firefox_ios_url('mozorg-firefox-home') }}" rel="external noopener" data-cta-type="link" data-cta-text="Download Browser for iOS">{{ get_ios }}</a></li>
+          </ul>
+        </div>
+        <p class="js-fx-only"><a class="mzp-c-cta-link" href="https://addons.mozilla.org/en-US/firefox/addon/facebook-container/{{ referrals }}">{{ get_extension }}</a></p>
+      {% endcall %}
+    </div>
+
+    <div class="mzp-l-content">
+        {% call feature_card(
+          title=monitor_title,
+          heading_level=3,
+          image_url='firefox/home/master/monitor.svg',
+          class='mzp-l-card-feature-right-half t-monitor',
+          link_url='https://monitor.firefox.com/' + referrals,
+          link_cta=monitor_cta,
+          ga_title='Monitor',
+          media_after=true
+        ) %}
+        {% endcall %}
+    </div>
+
+    <div class="mzp-l-content">
+      {% call feature_card(
+        title=lockwise_title,
+        heading_level=3,
+        image_url='firefox/home/master/lockwise.svg',
+        class='mzp-l-card-feature-left-half t-lockwise',
+        ga_title='Lockwise',
+        media_after=true
+      ) %}
+        <div class="mzp-c-menu-list mzp-t-cta mzp-t-download">
+          <h4 class="mzp-c-menu-list-title">{{ get_app }}</h4>
+          <ul class="mzp-c-menu-list-list" id="menu-lockwise">
+            <li class="mzp-c-menu-list-item"><a href="{{ lockwise_adjust_url('android', 'firefox-home') }}" rel="external noopener" data-cta-type="link" data-cta-text="Download Lockwise for Android"> {{ get_android }}</a></li>
+            <li class="mzp-c-menu-list-item"><a href="{{ lockwise_adjust_url('ios', 'firefox-home') }}" rel="external noopener" data-cta-type="link" data-cta-text="Download Lockwise for iOS">{{ get_ios }}</a></li>
+          </ul>
+        </div>
+        <p class="js-fx-only"><a class="mzp-c-cta-link" href="https://lockwise.firefox.com/{{ referrals }}" rel="external noopener" data-cta-type="link" data-cta-text="Lockwise Learn More">{{ lockwise_cta }}</a></p>
+      {% endcall %}
+    </div>
+
+    <div class="t-promise mzp-t-dark">
+      <div class="mzp-l-content">
+          {% call feature_card(
+            title=promise_title,
+            heading_level=3,
+            image_url='firefox/home/master/spacer.gif',
```

Comment:
spacer.gif ftw! :)

This is totally not a blocker, but I wonder if this should be a one off (non-protocol) component, since the final result is pretty far removed from a feature card?

Reply:
The appearance is far but the number of layout changes are really minimal so I felt like it was still appropriate.